tests/model_inheritance_regress/tests.py

```diff
@@ -186,8 +187,13 @@ def test_issue_7488(self):
         # Regression test for #7488. This looks a little crazy, but it's the
         # equivalent of what the admin interface has to do for the edit-inline
         # case.
-        suppliers = Supplier.objects.filter(
-            restaurant=Restaurant(name='xx', address='yy'))
+        # TODO: Change to assertRaises after the deprecation period.
+        with self.assertWarnsMessage(
+            RemovedInDjango40Warning,
+            'Passing an unsaved model object to related filter is deprecated.',
+        ):
+            suppliers = Supplier.objects.filter(
+                restaurant=Restaurant(name='xx', address='yy'))
```

Comment:
Have you checked ticket-7488? If admin filters by nonexistent objects then we need to fix this in advance.

Reply:
The ticket is obsolete and issue in admin doesn't exist.